psycopg2cffi/_impl/connection.py

```diff
@@ -121,7 +121,13 @@ def __init__(self, dsn, async=False):
         # The number of commits/rollbacks done so far
         self._mark = 0
 
-        self._async = async
+        if 'async' in kwargs:
+            self._async = kwargs.pop('async')
+        if 'async_' in kwargs:
+            self._async = kwargs.pop('async_')
+        else:
+            self._async = False
```

Comment:
this will overwrite passed ``async`` value (if ``async`` is passed by ``async_`` isn't, ``self._async`` will always be False